k8s/release/admin/secrets-csi-driver/patches/ptl/common/secrets-csi-driver.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: helm.fluxcd.io/v1
```

Comment:
you don't need to create multiple patches just put it in a nonprod folder and refer all envs to that